routes/manager/bookingApprove.js

```diff
@@ -0,0 +1,35 @@
+require("express-async-errors");
+const express = require("express");
+const { Booking } = require("../../models/booking");
+const router = express.Router();
+
+router.post("/:id", async (req, res) => {
+  const booking = await Booking.findByIdAndUpdate(
+    req.params.id,
+    {
+      $set: {
+        isApproved: true,
+      },
+    },
+    { new: true }
+  );
+  await booking.save();
+
+  const filter = {
+    roomId: booking.roomId,
```

Comment:
Please make sure that filtering gets all booking requests which have the same `arrivalDate` and `departureTime` without depending on the `roomId` as a criterion.